coding-the-legislation/30_case_disjunction.md

```diff
@@ -100,3 +100,7 @@ def formula(person, period):
 ```
 
 It is considered a good practice to always use helpers `where` and `select` when they are relevant, and not to emulate their behaviour manually with logical operations.
+
+## Using a dynamic parameter calculation
+
+I your want to implement a case disjunction where a result depends on a variable which can take only a finite amount of values, you should check the [Computing a parameter that depends on a variable](legislation_parameters.md#computing-a-parameter-that-depends-on-a-variable) section.
```

Comment:
If the result of your formula depends on a variable that can take only a finite amount of values, check out the [Computing a parameter that depends on a variable](legislation_parameters.md#computing-a-parameter-that-depends-on-a-variable) section.